jiant/trainer.py

```diff
@@ -436,6 +435,9 @@ def get_sampling_weights(
             sample_weights = 1 / task_n_train_examples
         elif weighting_method == "inverse_log_example":
             sample_weights = 1 / np.log(task_n_train_examples)
+        elif "examples-proportional-mixing" in weighting_method:
```

Comment:
For consistency with the other `weighting_method` names, please use underscores.